packages/components/src/components/post-breadcrumb-item/post-breadcrumb-item.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Element, h, Host, Prop, Watch } from '@stencil/core';
+import { version } from '@root/package.json';
+import { checkEmptyOrUrl } from '@/utils';
+
+/**
+ * @slot default - Slot for placing the text inside the breadcrumb link.
+ */
+
+@Component({
+  tag: 'post-breadcrumb-item',
+  styleUrl: 'post-breadcrumb-item.scss',
+  shadow: true,
+})
+export class PostBreadcrumbItem {
+  @Element() host: HTMLPostBreadcrumbItemElement;
+
+  /**
+   * The URL to which the breadcrumb item will link.
+   */
+  @Prop() readonly url!: string;
+
+  @Watch('url')
+  validateUrl() {
+    checkEmptyOrUrl(this.url, 'The "url" property is invalid');
```

Comment:
If the URL is mandatory:

```suggestion
    checkUrl(this.url, 'The "url" property is invalid');
```